imports/Api.app-test.js

```diff
@@ -6,13 +6,106 @@ import fetch from 'isomorphic-unfetch';
 // the test runner starts the server tests before the database is fully
 // populated. As a result those tests will very likely time out.
 if (Meteor.isClient) {
-	describe('GroupApi', function () {
-		it('should return JSON response', function (done) {
-			const groups = Meteor.absoluteUrl('/api/0/json/groups');
-			fetch(groups).then((result) => {
-				expect(result.status).to.equal(200);
-				expect(result.headers.get('Content-Type')).to.equal('application/json; charset=utf-8');
-				done();
+
+	describe('Api', function () {
+		this.timeout(15000);
+		describe('GroupApi', function () {
+			describe('Get all groups', function(){
+				it('should return JSON response', function (done) {
+					const groups = Meteor.absoluteUrl('/api/0/json/groups');
+					fetch(groups).then((result) => {
```

Comment:
I realize this configuration was already in the old test-code. So this is just for your information. Here goes:

The `future` constructed by `fetch` must be returned from the test-case, and the `it`-callback must not take the `done` parameter. Otherwise if an error occurs in the async parts this is not reported by `chai`.